cicd-utils/info.json

```diff
@@ -40,8 +41,9 @@
       "operation": "split_export_templates",
       "annotation": "split_export_templates",
       "category": "utilities",
-      "description": "Split the export template recordset into Production Content, Production Settings and Development Settings export template recordset",
+      "description": "Split the export template file into Production Content, Production Settings and Development Settings export template files",
```

Comment:
Suggestion: Split the export template file into three export templates named Production Content, Production Settings and Development Settings

Reply:
done